config/jobs/image-pushing/k8s-staging-kubebuilder.yaml

```diff
@@ -0,0 +1,24 @@
+postsubmits:
+  kubernetes-sigs/kubebuilder:
+
+    - name: post-kube-rbac-proxy-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-kubebuilder-kube-rbac-proxy, sig-k8s-infra-gcb
+      decorate: true
+      run_if_changed: 'TAG'
+      branches:
+        - ^kube-rbac-proxy-releases$
+      spec:
+        serviceAccountName: gcb-builder
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/image-builder:v20230111-cd1b3caf9c
+            command:
+              - /run.sh
+            args:
+              - --project=k8s-staging-kubebuilder
+              - --scratch-bucket=gs://k8s-staging-kubebuilder-gcb
+              - --env-passthrough=PULL_BASE_REF
+              - --with-git-dir
+              - --gcb-config=build/cloudbuild_kube-rbac-proxy.yaml
+              - .
```

Comment:
suggestion: instead of using --gcb-config here, probably you want to create a new directory in https://github.com/kubernetes-sigs/kubebuilder/tree/kube-rbac-proxy-releases with a proper cloudbuild, and point to that directory

eg.:
```
args:
....
  - images/
```

And add a new cloudbuild.yaml inside this directory (see https://github.com/kubernetes/ingress-nginx/blob/main/cloudbuild.yaml). In this case if you want to test different combinations of cloudbuild.yaml later you won't need to change the one that is already working.

Reply:
ok I see it here: https://github.com/kubernetes-sigs/kubebuilder/pull/3362/files

let me continue the discussion there

Reply:
So, I understand that is right. Am I right?

Reply:
+1 for this suggestion, but not blocking
put a hold in case